integr8ly/tutorial-web-app
# INTLY-1852: Add error page

## Motivation
[INTLY-1852](https://issues.jboss.org/browse/INTLY-1852)

## What
Add 404 error page when attempting to navigate to a non-existent Solution Explorer page

## How
Add a redirect to the 404 error page for any URI that does not match an existing route that exists in routes.js. Also resolved a console error in which an array was specified for the To prop of the home page, but it only accepts strings in Router.js.

## Verification Steps
1. Open the Solution Explorer home page.
2. Append **/string** to the home page, where **string** does not equal an existing page like 'settings'. For example, /iloveredhat.
3. Verify that the 404 error page displays, and that you can click the 'Return to home' button to get back to the home page.

## Checklist:

## Progress

## Additional Notes
Should be fine to test these changes locally, but feel free to test on my server:
https://tutorial-web-app-webapp.apps.uxddev-17f0.openshiftworkshop.com

Or update the tutorial-web-app deployment on your own server with a docker image of these changes:
docker.io/mfrances17/dev-tutorial-web-app:latest

Screen cap:
![404_on_server](https://user-images.githubusercontent.com/39063664/62079145-14d76680-b21c-11e9-8737-53ef882ba72d.png)
